packages/embark-ui/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.1](https://github.com/embark-framework/embark/compare/v4.0.0...v4.0.1) (2019-03-26)
+
+**Note:** Version bump only for package embark-ui
```

Comment:
I was confused why no commits are in the log (didn't notice previously). See: https://github.com/embark-framework/embark/compare/30e5af5038978439863040cf5056c85f843d89df...4.0.x.

I guess it's because refactor commits aren't tracked in the changelog.

Reply:
Yes we'd need to update the range there as well (manually)

Reply:
@michaelsbradleyjr I'm sorry I completely misunderstood. Wasn't focusing.

So yes, refactor() commits are not parsed into the changelog because they are neither a fix, nor feature (or breaking change). If something is a refactor it shouldn't affect public APIs.

If it does, it's already a breaking change and then it will show up in the changelog.

Rolling back on this one!